src/Pages/signin/Containers/SignInContainer.tsx

```diff
@@ -1,18 +1,60 @@
 import React from "react";
-import SignInForm from "../Components/SignInForm";
-import Box from "@material-ui/core/Box";
+import {
+  StyledContainer,
+  StyledFormArea,
+  Logo,
+  StyledFormButton
+} from "../Components/SignInForm";
+import { makeStyles } from "@material-ui/core/styles";
+import { TextField } from "@material-ui/core";
+
+const useStyles = makeStyles(() => ({
+  form: {
+    width: "80%",
+    marginLeft: "10%",
+    marginRight: "10%",
+  },
+}));
 
 const SignInContainer: React.FC = () => {
+  const classes = useStyles();
   return (
-    <Box
-      display="flex"
-      justifyContent="center"
-      alignItems="center"
-      width="100%"
-      height="100vh"
-    >
-      <SignInForm />
-    </Box>
+    <>
+      <StyledContainer>
```

Comment:
Ahh, this is a good coding style but you place the code wrong folder hahaha. Please move lines 23 to 56 to SignInForm.js and replace this line with `<SignInForm />` that is imported from SignInForm.js.

Reply:
- [ ] Move codes.